src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -96,20 +96,85 @@ void createStationWithDuplicateName() {
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
-     * Then 2개의 지하철역을 응답 받는다
+     * Then 지하철역 목록을 응답 받는다
+     * Then 생성한 지하철역 2개가 응답받은 지하철역 목록에 존재한다
      */
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        // given
+        String[] createStations = new String[] {"강남역", "교대역"};
+        for (String createStation : createStations) {
+            createStationForTest(createStation);
+        }
```

Comment:
```suggestion
        지하철역_생성됨(["강남역","교대역"]);
```

이렇게 해볼수도 있겠네요!
테스트 코드는 가독성이 중요하다고 생각해요!